test_crates/modules/src/lib.rs

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
minor nit: I'd just strip the leading and trailing whitespace in both of these test files, except for a single trailing newline at the end of all files